### PR TITLE
Fix #17

### LINKED ISSUE #17: Quotes around attributes are stripped
After using setAttribute to update the href of a link the double quotes are stripped from the attributes when converting it back to a string using toString. 

Example:

Original: `<a style="width:100%;" href="https://google.com" class="button--action button-text-wrap" target="_blank" rel="noopener noreferrer">Button</a>`

New: `<a style=width:100%; href=http://localhost:8080/metrics/link?redirect=https://google.com&#x26;subject=TEST%20-%20TEST%20-%20test%20link&#x26;name=Button class=ods-button--action button-text-wrap target=_blank rel=noopener noreferrer>Button</a>`

Notice the double quotes around the attributes are now gone.

### PR DESCRIPTION
**Pull request by Sharcoux**

### CODE REVIEW

File: src/nodes/html.ts [hunk 1]
```diff
@@ -465,49 +465,43 @@ export default class HTMLElement extends Node {
 	/**
 	 * Set an attribute value to the HTMLElement
 	 * @param {string} key The attribute name
-	 * @param {string} value The value to set, or null / undefined to remove an attribute
+	 * @param {string} value The value to set
 	 */
-	public setAttribute(key: string, value: string) {
-		if (arguments.length < 2) {
-			throw new Error('Failed to execute \'setAttribute\' on \'Element\'');
+	setAttribute(key: string, value: string) {
```
[the remark this answers is not in the document]
Comment (Sharcoux) [reply src/nodes/html.ts]: The behaviour is the same : it throws an error. The current error message is just terrible and useless. What's the issue with giving more details?

**Comment by nonara** [comment]
Doing a little housekeeping. Is this PR something that still needs to be reviewed?

**Comment by Sharcoux** [comment]
Probably better to report the changes than review. The version is too old. The only thing worth doing here is check the error thrown by setAttributes if the parameters are wrong and see if it's helping in case of debugging. Otherwise, this can be closed.

**Comment by Sharcoux** [comment]
I kept using my fork actually. I liked more the previous architecture ;)

**Pull request closed without merging**